packages/core/src/test/codewhisperer/commands/basicCommands.test.ts

```diff
@@ -426,10 +400,10 @@ describe('CodeWhisperer-basicCommands', function () {
         it('shows expected quick pick items when connected', async function () {
             sinon.stub(AuthUtil.instance, 'isConnectionExpired').returns(false)
             sinon.stub(AuthUtil.instance, 'isConnected').returns(true)
-            sinon.stub(CodeScansState.instance, 'isScansEnabled').returns(false)
+            await CodeScansState.instance.setScansEnabled(false)
             getTestWindow().onDidShowQuickPick((e) => {
                 e.assertContainsItems(
-                    createAutoSuggestions(false),
+                    createAutoSuggestions(true),
```

Comment:
`true` is the default since https://github.com/aws/aws-toolkit-vscode/commit/1bec3c3eeba9b2d962db8abbd701bda28dae103e . This wasn't caught because the tests use stubs/mocks. This is an example of why stubs/mocks are fragile and don't represent the actual application code.